docs/development.rst

```diff
@@ -1183,3 +1183,20 @@ HTML output in the ``_build/html/`` directory.
 .. note::
 
     You will need ``stdpopsim`` to be installed for the build to work.
+
+
+********************
+Making a new release
+********************
+
+Here is a list of things to do when making a new release
+(TODO: add to this):
+
+- increment the version number in ``stdpopsim/_version.py``
```

Comment:
This shouldn't be needed. This file will be updated by `setuptools_scm` from the git tag. So just committing/pushing the new version tag should be sufficient (and might trigger a rebuild in the ci labyrinth...? I forget).

Reply:
This is something setuptools_scm recommends *not* doing

Reply:
ah, ok! =)